content/applications/finance/accounting/taxation/taxes/vat_units.rst

```diff
@@ -0,0 +1,86 @@
+=========
+VAT units
+=========
+
+.. important::
+   This is only applicable to multi-company environments.
+
+A **VAT unit** is a group of VAT-taxable enterprises that are legally independent of each other but
+are closely linked financially, organizationally, and economically and therefore considered the same
+VAT-taxable enterprise. **VAT units** are not mandatory, but if created, constituent companies of
+the unit must belong to the same **country**, use the same **currency**, and one company must be
+designated as the **representative** company of the **VAT unit**. **VAT units** receive a specific
+**tax ID** intended only for **tax returns**. **Constituent** companies keep their **tax ID** used
+for **commercial purposes**.
+
+.. example::
+   Enterprise **A** owes 300.000,00€ of VAT taxes and enterprise **B** can recover 280.000,00€ of
+   VAT taxes. They form up as a **VAT unit** so that the two amounts balance out and must conjointly
+   only pay 20.000,00€ of VAT taxes.
+
+Configuration
+=============
+
+To configure a **VAT unit**, go to :menuselection:`Settings --> General Settings`, scroll down to
+the :guilabel:`Companies` section, and click :guilabel:`Manage Companies`. Select the company to
+serve the **representative** role, and in the :guilabel:`General Information` tab, fill in the
+mandatory fields for the .XML export when exporting the **tax report**: :guilabel:`Company Name`,
+:guilabel:`Address`, :guilabel:`VAT`, :guilabel:`Currency`, :guilabel:`Phone`, and
+:guilabel:`Email`.
+
+.. image:: vat_units/vat-unit-general.png
+   :align: center
+   :alt: General information tab
+
+Then, click on the :guilabel:`VAT Units` tab, :guilabel:`Add a line`, and either select an existing
+**VAT unit**, or :guilabel:`Create` a new one. Enter a **name** for the unit, :guilabel:`Country` of
```

Comment:
As in the other PR, I think we shouldn't use guilabels when we are not referring to an element on screen.
```suggestion
**VAT unit**, or create a new one. Enter a **name** for the unit, :guilabel:`Country` of
```